toast-tk-web-agent/src/main/java/io/toast/tk/agent/web/rest/StopHandler.java

```diff
@@ -20,14 +22,20 @@ public StopHandler(BrowserManager browserManager, IAgentServer agentServer) {
 	}
 
 	@Override
-	public void handle(HttpServerRequest req) {
+	public void handle(RoutingContext rc) {
 		if (browserManager.getDriver() != null) {
 			browserManager.getDriver().close();
 		}
-		agentServer.unRegister();
-		req.response().headers().add("Access-Control-Allow-Origin", "*");
-		req.response().setStatusCode(200).end();
-		System.exit(0);
+		try {
+			agentServer.unRegister();
+		} catch (UnknownHostException e) {
+			e.printStackTrace();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1148)